addons/dispatch/1.4.x/dispatch-3.yaml

```diff
@@ -85,6 +85,8 @@ spec:
       dispatch:
         image: "mesosphere/dispatch:1.4.0"
         testImage: "mesosphere/dispatch-helm-test:1.4.0"
+      kommander-ui:
+        defaultTag: 6.84.2
```

Comment:
the usage is `:{{ default (tpl .Values.image.defaultTag .) .Values.image.tag }}` in the [kommander-ui chart](https://github.com/mesosphere/kommander-ui/blob/master/chart/kommander-ui/templates/deployment.yaml#L41). 

Should this be:
```
kommander-ui:
   image:
      defaultTag: 6.84.2
```

Reply:
oops. thanks.